packages/astro/src/core/config/schema.ts

```diff
@@ -13,6 +13,7 @@ const ASTRO_CONFIG_DEFAULTS: AstroUserConfig & any = {
 	srcDir: './src',
 	publicDir: './public',
 	outDir: './dist',
+	cacheDir: './node_modules/.astro',
```

Comment:
Why not the top-level `.astro` folder?

Reply:
This would require users to set up caching manually on their host, while `node_modules` is already cached by most hosts (same reason why Vite's cache is in `node_modules`, as far as I know)

Reply:
Ok, I didn't realize this. Makes sense in that case.